src/sql/run.py

```diff
@@ -184,11 +184,12 @@ def DataFrame(self, payload):
         return frame
 
     @telemetry.log_call("polars-data-frame")
-    def PolarsDataFrame(self):
+    def PolarsDataFrame(self, polars_dataframe_kwargs=None):
```

Comment:
```suggestion
    def PolarsDataFrame(self, **polars_dataframe_kwargs):
```

making this keyword arguments will lead to a cleaner API.

e.g.,

```python
result.PolarsDataFrame(a=1, b=2)
```